scionlab/scion_config.py

```diff
@@ -123,6 +123,9 @@ def _gen_configs(self, cb):
 
         for service in self._control_services():
             if service.type == Service.CS:
+                if service.AS.is_infrastructure_AS():
```

Comment:
>if service.AS.is_infrastructure_AS():

Is there a reason we are special casing this for infrastructure ASes?

If UserASes want a different configuration for some reason, they can still overwrite it, right?

Reply:
The disallow ISD loops inside the beaconing policy is necessary in the infrastructure nodes only. User ASes behave per default when beaconing, and they can of course modify their beaconing policy whenever they want. I don't have a strong opinion on limiting this policy to infrastructure ASes only, but I didn't see a reason to do have it also in user ASes.

Reply:
>didn't see a reason to do have it also in user ASes

To not special case them (, and later having to revert it when introducing UserAS APs)